### ISSUE TITLE
Game crash when joining a server

### ISSUE THREAD

**Issue by vos6434** [momo5502/boiii#132]
Game crashed when I joined the test server.
Crashlog: [boiii-crash-2023-02-10-11-34-23.zip](https://github.com/momo5502/boiii/files/10706961/boiii-crash-2023-02-10-11-34-23.zip)

**Comment by FrancescoCode1** [comment]
Getting the same error. Always when trying to join a dedicated server. Offline game works fine. Tested using unofficial cracked BO3 and using "clean" files from steam.
Heres the dump: 
[boiii-crash-2023-02-10-19-37-01.zip](https://github.com/momo5502/boiii/files/10711189/boiii-crash-2023-02-10-19-37-01.zip)

**Comment by Alucard935** [comment]
![image](https://user-images.githubusercontent.com/107876885/218518535-daefe319-5823-461f-bc7a-de8b58d0aa2e.png)
I have the same problem when i try to join to a server alway crash and this is the message

**Comment by momo5502** [comment]
Pretty sure it's fixed by this: 7a6d5b71122cc9fcd510b772f1500ce93e4831cb
If not, please reopen

**Comment by Alucard935** [comment]
![image](https://user-images.githubusercontent.com/107876885/218595221-bdbf09fc-c8e9-41ad-b3dd-cecc626f7c63.png)
Still crashing me i don't know already comprobe the game files and all it's ok. Just crash when i try to join to a server and i still hearing de music of the menu and i have to close the game with the task manager

**Comment by momo5502** [comment]
> ![image](https://user-images.githubusercontent.com/107876885/218595221-bdbf09fc-c8e9-41ad-b3dd-cecc626f7c63.png)
> Still crashing me i don't know already comprobe the game files and all it's ok. Just crash when i try to join to a server and i still hearing de music of the menu and i have to close the game with the task manager

You issue is different to the crash in the initial report. Seems like your gamefiles are simply broken. Go verify them through steam.

**Comment by FrancescoCode1** [comment]
In my case the crash was caused by steam not running (most of the time). Though the same fatal error appeared quite a few times while steam was running. But joining servers is always a gamble for me anyway, im going to create another issue for that.

**Comment by momo5502** [comment]
> In my case the crash was caused by steam not running (most of the time). Though the same fatal error appeared quite a few times while steam was running. But joining servers is always a gamble for me anyway, im going to create another issue for that.

Exactly that has been fixed. Don't open an issue unless you can reproduce it again with the latest binary
